core/src/main/scala/spinal/core/formal/SymbiYosysBackend.scala

```diff
@@ -29,8 +29,8 @@ sealed trait SbyEngine extends FormalEngin {
 object SmtBmcSolver extends Enumeration {
   type SmtBmcSolver = Value
 
-  val Yices = Value("yices")
-  val Boolector = Value("booleactor")
+  val Yices = Value("yices")      // Yices2 ?
```

Comment:
I have tested the code with yices, it is named without version code. So it is correct.